src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6841,6 +6841,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureCollectionExpressions" xml:space="preserve">
     <value>collection expressions</value>
   </data>
+  <data name="IDS_FeatureCollectionExpressionArguments" xml:space="preserve">
```

Comment:
Why do we want a new feature flag? Seems `IDS_FeatureDictionaryExpressions` would be sufficient

Reply:
The motivating scenario for collection arguments is dictionary key comparer, but collection arguments can be used with non-dictionary types. The additional id allows the feature to be reported "collection expression arguments" rather than "dictionary expressions" for the error reported with earlier language versions.